src/generate.ts

```diff
@@ -59,35 +29,36 @@ export async function generate(opts: GenerateOpts): Promise<void> {
  * Callers can optionally pass in their own `sectionName -> RuleFn` `ruleFns` but we'll also default
  * to the out-of-the-box Tachyons-ish rules defined in `defaultRuleFns`.
  */
-export function generateRules(
-  config: TrussConfig,
-  ruleFns?: Record<string, RuleFn>
-): Record<string, string[]> {
+function generateRules(
```

Comment:
This is no longer public b/c the application's `truss/index.tsx` doesn't have to call it anymore, it can pass in `section -> rulefn`s to be invoked as part of the config object.